generated/datamigration/microsoft.datamigration/2021-06-30/types.md

```diff
@@ -1082,7 +1116,17 @@
 
 ### MigrateSqlServerSqlDbTaskOutputDatabaseLevelValidationResult
 #### Properties
+* **dataIntegrityValidationResult**: [DataIntegrityValidationResult](#dataintegrityvalidationresult) (ReadOnly): Provides data integrity validation result between the source and target tables that are migrated.
+* **endedOn**: string (ReadOnly): Validation end time
+* **id**: string (ReadOnly): Result identifier
+* **migrationId**: string (ReadOnly): Migration Identifier
+* **queryAnalysisValidationResult**: [QueryAnalysisValidationResult](#queryanalysisvalidationresult) (ReadOnly): Results of some of the query execution result between source and target database
 * **resultType**: 'MigrationDatabaseLevelValidationOutput' (Required): Result type
+* **schemaValidationResult**: [SchemaComparisonValidationResult](#schemacomparisonvalidationresult) (ReadOnly): Provides schema comparison result between source and target database
+* **sourceDatabaseName**: string (ReadOnly): Name of the source database
+* **startedOn**: string (ReadOnly): Validation start time
+* **status**: 'Completed' | 'CompletedWithIssues' | 'Default' | 'Failed' | 'InProgress' | 'Initialized' | 'NotStarted' | 'Stopped' | string (ReadOnly): Current status of validation at the database level
+* **targetDatabaseName**: string (ReadOnly): Name of the target database
```

Comment:
This alteration has occurred because `MigrateSqlServerSqlDbTaskOutputDatabaseLevelValidationResult` uses [multiple inheritance to define its properties](https://github.com/Azure/azure-rest-api-specs/blob/eeb2cff50119cfa7b724ac63c768d637ec01827a/specification/datamigration/resource-manager/Microsoft.DataMigration/stable/2021-06-30/definitions/MigrateSqlServerSqlDbTask.json#L364-L375). All added properties are defined on `MigrationValidationDatabaseLevelResult`.